pyoverkiz/enums/state.py

```diff
@@ -14,15 +14,16 @@ class OverkizAttribute(str, Enum):
 class OverkizState(str, Enum):
     """Device states used by Overkiz."""
 
-    CORE_ACTIVE_ZONES = "core:ActiveZonesState "
+    CORE_ACTIVE_ZONES = "core:ActiveZonesState"
```

Comment:
Good catch!

Reply:
Can you maybe fix the ventilation mode as well? (in this PR or another one :D)

Reply:
What’s the issue with`    IO_VENTILATION_MODE = "io:VentilationModeState"`
?

Reply:
SET_VENTILATION_MODE (command) has a typo in its enum

Reply:
there was a typo, but looks fixed already

Reply:
https://github.com/iMicknl/python-overkiz-api/blob/837f019b88d5f6c1f0e16ac981489d5ef87783d5/pyoverkiz/enums/command.py#L73